### DISCUSSION TITLE
Nucleosynthesis and element abundance - "Defying the Big Bang"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#284] [Astrophysics and Physics]
"Defying the Big Bang: The Elements That Don’t Fit"
https://youtu.be/hovNhrIG-Ko

Has a good review of the difficulties in explaining element abundances, both for ΛCDM and plasma cosmology.

**Comment by LifeIsStrange** [comment]
See also: https://en.wikipedia.org/wiki/P-nuclei there aren't just issues with BBN nucleosynthesis but also with stellar nucleosynthesis.
(see also stars older than the universe)
It is also often forgotten that light isn't everything and that the chemical flux of cosmic rays that comes to us is anomalous.

**Comment by RedshiftDrift** [comment]
"Unexpected shape of lead-208 nucleus prompts reevaluation of atomic nuclei models"

> "The discovery challenges fundamental assumptions about nuclear structure and has far-reaching implications for our understanding of how the heaviest elements are formed in the universe."

https://phys.org/news/2025-02-unexpected-nucleus-prompts-reevaluation-atomic.html

> "Thus, in spite of its well-studied nature, $^{208}$ Pb remains a puzzle for our understanding of nuclear structure."

https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.134.062502

**Comment by RedshiftDrift** [comment]
["Oxygen discovered in most distant known galaxy"](https://www.eso.org/public/news/eso2507/)
"the galaxy is much more chemically mature than expected..." in $\Lambda\mbox{CDM}$.

That galaxy, JADES-GS-z14-0, is perfectly normal when considered from outside the Big Bang paradigm.

**Reply by LifeIsStrange** [reply]
See also HN comments
https://news.ycombinator.com/item?id=43422909
> To put this in persepctive, our Sun, which is larger than the median star, produces no oxygen and is ~4.5 billion years old.

Larger stars are theorized to undergo the so-called CNO cycle [1]. To get to this process after "only" 300 million years means the stars are truly massive.

[1]: https://en.wikipedia.org/wiki/CNO_cycle
